### PR TITLE
Added new test functionality in cpu:em_smt_folding_test

### PR DESCRIPTION
**Pull request by PraveenPenguin**
1- Added cpu dlapr operation code which does cpu hot remove
and hot add
2- call main test as well dlpar cpu operation in thread

Signed-off-by: Praveen K Pandey <praveen@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by PraveenPenguin** [comment]
job run 
```[root@system ~]# avocado run em_smt_folding_test.py 
Traceback not available
JOB ID     : e9174a1c07c7e6a539685f1d4fe614390689c9b3
JOB LOG    : /home/workspace/runAvocadoFVTTest/avocado-fvt-wrapper/results/job-2019-03-29T10.27-e9174a1/job.log
 (1/1) em_smt_folding_test.py:SmtFolding.test: PASS (17.98 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 18.34 s
JOB HTML   : /home/workspace/runAvocadoFVTTest/avocado-fvt-wrapper/results/job-2019-03-29T10.27-e9174a1/results.html
[root@system ~]# 
````

File: cpu/em_smt_folding_test.py [hunk 1]
```diff
@@ -71,7 +73,41 @@ def test(self):
...
+    def run_ebizzy(self):
+        '''
+        Run ebizzy by doing taskset
         '''
+        output = process.system_output("taskset -c %s ./ebizzy -t1"
+                                       " -S 6 -s 4096" % self.cpu, shell=True)
+        return output.split()[0]
+
+    def dlpar_cpu_hotplug(self):
+        if 'ppc' in platform.processor() and 'PowerNV' not in open('/proc/cpuinfo', 'r').read():
```
Comment (nasastry) [inline cpu/em_smt_folding_test.py]: in `setUp` `ppc` part was already checked and it's not required here.
```
        if 'ppc' not in distro.detect().arch:
            self.cancel("Processor is not ppc64")
```
and for reading `/proc/cpuinfo` please use `genio.read_file`

Comment (PraveenPenguin) [reply cpu/em_smt_folding_test.py]: @nasastry ah sure , will make changes and push

**Comment by PraveenPenguin** [comment]
@nasastry pushed the changes can you please review

**Comment by narasimhan-v** [comment]
@nasastry Thanks.

**Pull request merged**